extension/scripts/content/items/ttItems.js

```diff
@@ -6,552 +6,608 @@ let pendingActions = {};
 	await loadDatabase();
 	console.log("TT: Items - Loading script. ");
 
-	storageListeners.settings.push(loadItems);
-	loadItems();
+	let itemsFeatures = new ItemsFeatures();
+	itemsFeatures.init();
 
-	loadItemsOnce();
+	storageListeners.settings.push(function () {
+		itemsFeatures.init();
+		featureManager.display(settings.featureDisplay);
```

Comment:
Isn't this handled in the ttGlobal.entry already?

Reply:
Yep, my bad. Fixed.